Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu
+FROM ubuntu:focal
```

Comment:
> Fix the ubuntu image to focal to match the current deployment to shinyapps.io.

How do you know that shinyapps.io uses `ubutu:focal`?  Is that documented somewhere?  Is there some assurance it won't change?

Reply:
https://docs.posit.co/shinyapps.io/appendix.html#default-system-packages. I don't know what their plans are to update that service. They had no plans to add support for python3.10 if that's any indicator.